cpp/include/cuml/linear_model/ols_mg.hpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2020 NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cuml/linear_model/glm.hpp>
+#include <opg/matrix/data.hpp>
+#include <opg/matrix/part_descriptor.hpp>
+
+#include <common/cumlHandle.hpp>
+
+namespace ML {
+namespace OLS {
+namespace opg {
+
+/**
+ * @brief performs MNMG fit operation for the ridge regression
+ * @input param handle: the internal cuml handle object
```

Comment:
Our nightly build has been enforcing proper doxygen format. These should be `@param[in]` or `@param[out]`. This can be verified locally with `build.sh cppdocs`. I just updated k-means, pca/tsvd/sign-flip, and knn. We should probably update this before merging.